plotting/plotdataMC.py

```diff
@@ -364,5 +438,5 @@
         if arg.norm:scale="_norm"
         name="all"
         if not arg.combine:name=str(j)
-        if(arg.log):fig.savefig("plot/%s_unc_%s_inclusive%s_%s_%s.pdf" %(arg.phase, discrs, scale, arg.ext,name))
-        else:fig.savefig("plot/%s_unc_lin_%s_inclusive%s_%s_%s.pdf" %(arg.phase, discrs, scale, arg.ext,name))
+        if(arg.log):fig.savefig("%s_unc_%s_inclusive%s_%s_%s.pdf" %(arg.phase, discrs, scale, arg.ext,name))
```

Comment:
```suggestion
    if(arg.log):
        fig.savefig(f"{arg.phase}_unc_{discrs}_inclusive{scale}_{arg.ext}_{name}.pdf")
```
It's probably safe to assume this will be run in `3.7+` so you can use f-strings

Reply:
Done